docs/troubleshooting.md

```diff
@@ -0,0 +1,54 @@
+# Trouble shooting
+
+This section is meant to provide guidance to recover from the most known issues you might run into. 
+
+### Did `deliver` but meant `deliver --local`
+read [deliver]() docs to understand the command
+##### Experience
+the work is gone and you cannot deliver the work again.
+
+##### Issue
+Deliver pushes the work to the remote repository on a `ready/name` branch, so it can be integrated if a server have been configured. 
+phlow will not deliver branches prefixed with `delivered` so you cannot re-deliver it. 
+
+##### Solution
+1. rename the delivered branch
+2. checkout the branch
+3. deliver your work again, nothing should stop you. 
+4. delete the remote branch
+
+```sh
+git branch -m delivered/10-my-issue 10-my-issue #1
+git checkout 10-my-issue    #2
+git phlow deliver --local   #3
+git push -d origin ready/10-my-issue #4
+```
+
+### Did `deliver` before `wrapup`
+read [deliver]() docs to understand the command
+##### Experience 
+The workspace is still dirty and your changes have not been added. 
+##### Issue
+`deliver`pushes a branch to GitHub but it is empty because you work was not comitted. phlow will not deliver branches prefixed with `delivered` so you cannot re-deliver it. 
```

Comment:
space after `deliver`.
you work -> your work
phlow -> git-phlow.
This also feels like it is something git-phlow should be able to handle gracefully. I suggest an issue for this case